packages/block-library/src/image/index.php

```diff
@@ -252,6 +256,14 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 	$processor->set_attribute( 'data-wp-init', 'callbacks.setButtonStyles' );
 	$processor->set_attribute( 'data-wp-on--load', 'callbacks.setButtonStyles' );
 	$processor->set_attribute( 'data-wp-on-window--resize', 'callbacks.setButtonStyles' );
+
+	// Set an event to preload the image on pointerenter and pointerdown(mobile).
+	// Pointerleave is used to cancel the preload if the user hovers away from the image
+	// before the predefined delay.
+	$processor->set_attribute( 'data-wp-on--pointerenter', 'actions.preloadImageWithDelay' );
+	$processor->set_attribute( 'data-wp-on--pointerdown', 'actions.preloadImage' );
+	$processor->set_attribute( 'data-wp-on--pointerleave', 'actions.cancelPrefetch' );
```

Comment:
`actions.cancelPreload` instead?

```suggestion
	$processor->set_attribute( 'data-wp-on--pointerleave', 'actions.cancelPreload' );
```

Reply:
Good catch! Renamed to `cancelPreload` ✅